Release 0.0.12 (#30)
Regression-Games/RegressionDocs: docs/changelog.mdx

# Changelog

## 0.0.12

:::info

This version is a release preview. Our product is in beta and we would love for developers to try it out and provide feedback! Contact
us at info@regression.gg for dedicated support and access.

:::

**New Platform Features**

> - New RGAction attribute within RG Unity SDK to make action definitions easier to make

Reviewer: ```suggestion
- New RGAction attribute within RG Unity SDK to make actions easier to define
```

- [GitHub Actions and Unity Test Runner Support](/studios/unity/tutorials/github_actions) - New samples and guides for running bots 
  in GitHub Actions (and other CI/CD systems via [GameCI](https://game.ci))
- UI improvements to Behavior Tree Beta
- Updated defaults for RG Unity SDK to be more user-friendly
- The RG Unity SDK now includes its own font assets for better interoperability
- Links to download Zip bots from dashboard are now available

**Bug fixes**

- Fixed issue where the website would freeze in some match loading scenarios
- Fixed broken links in documentation
- Fixed issue where Agent Builder code generation would flicker
- Fixed bug where Unity actions would sometimes be sent to the wrong bots

> - Fixed bug where instant bots would not start when no GitHub account was included in a user account

Reviewer: ```suggestion
- Fixed bug where instant bots would not start when the user did not have a linked GitHub account
```
